orchestra/models.py

```diff
@@ -461,6 +461,43 @@ def __str__(self):
             str(self.task), self.assignment_counter, str(self.worker))
 
 
+class Iteration(models.Model):
```

Comment:
I really don't like this naming practice, so I was hoping to break it subtly :)

I would prefer that it be called iteration, but would prefer `AssignmentIteration` if you think we should continue to follow that convention.